draft-ietf-mls-protocol.md

```diff
@@ -2319,11 +2319,12 @@ member's public signing key. The `credential` field contains information
 authenticating both the member's identity and the provided signing key, as
 described in {{credentials}}.
 
-The `capabilities` field indicates what protocol versions, ciphersuites,
-extensions, credential types, and non-default proposal types are supported by a client.
-Proposal and extension types defined in this document are considered "default" and thus need
+The `capabilities` field indicates the protocol features that the client
+supports, including protocol versions, ciphersuites, credential types,
+non-default proposal types, and non-default extension types.
+Proposal and extension types defined in this document are considered "default" and thus MAY
 not be listed, while any credential types the application wishes to use MUST
```

Comment:
```suggestion
Proposal and extension types defined in this document are considered "default" and thus are not listed, while any credential types the application wishes to use MUST
```
If we don't have any reason to include default proposals and extensions, then they should never be included. There should be exactly one way to say that an MLS client implements the default.